Dynatrace/dynatrace-operator
# Implement conditions for EC config secret

## Description

This PR introduce new conditon type and project reconciliation reason of it.

ticket -> [DAQ-1388](https://dt-rnd.atlassian.net/browse/DAQ-1388)

## How can this be tested?

`make test/e2e/edgeconnect` or unit-test or manually via creating Edgeconnect CR and check condition type and status.

[DAQ-1388]: https://dt-rnd.atlassian.net/browse/DAQ-1388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
